CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Changelog
+## 1.0.0: September 30, 2022
```

Comment:
mmmm, did I miss something, or get trapped in a time warp somewhere :)

Reply:
Made an assumption that it was attempting to indicate v1.0.0 projected launch date as it was updated from the original date of ## 1.0.0: September 1, 2022... Is this meant to be the current date?